### ISSUE TITLE
Double notifications

### ISSUE THREAD

**Issue by liamfrager-revature** [BitsuRevature/social-media-fe#1]
"Loading Posts" notification is doubled when visiting the Home page. I'm guessing this is the same reason that React will run `useEffect()` multiple times on component load.

**Comment by BitsuRevature** [comment]
Yeah, likely. I tried to resolve it, but I couldn't find a way to.
